tests/e2e/testsLibrary/WorkspaceHandlingTests.ts

```diff
@@ -80,30 +80,36 @@ export class WorkspaceHandlingTests {
 
     public obtainWorkspaceNameFromStartingPage(): void {
         test('Obtain workspace name from workspace loader page', async() => {
-            try {
-                Logger.info('Waiting for workspace name on workspace loader page');
-                await this.driverHelper.waitVisibility(WorkspaceHandlingTests.READY_TO_READ_WORKSPACE_NAME_LOCATOR, TimeoutConstants.TS_WAIT_LOADER_PRESENCE_TIMEOUT);
+            Logger.info('Waiting for workspace name on workspace loader page');
+            await this.driverHelper.waitVisibility(WorkspaceHandlingTests.READY_TO_READ_WORKSPACE_NAME_LOCATOR, TimeoutConstants.TS_WAIT_LOADER_PRESENCE_TIMEOUT);
 
-                const timeout: number = TimeoutConstants.TS_IDE_LOAD_TIMEOUT;
-                const polling: number = TestConstants.TS_SELENIUM_DEFAULT_POLLING;
-                const attempts: number = Math.ceil(timeout / polling);
-                let startingWorkspaceLineContent: string;
+            const timeout: number = TimeoutConstants.TS_IDE_LOAD_TIMEOUT;
+            const polling: number = TestConstants.TS_SELENIUM_DEFAULT_POLLING;
+            const attempts: number = Math.ceil(timeout / polling);
+            let startingWorkspaceLineContent: string = '';
 
-                for (let i: number = 0; i < attempts; i++) {
+            for (let i: number = 0; i < attempts; i++) {
+                try {
                     startingWorkspaceLineContent = await this.driverHelper.getDriver().findElement(WorkspaceHandlingTests.START_WORKSPACE_PAGE_NAME_LOCATOR).getAttribute('innerHTML');
```

Comment:
We can rewrite this method to:
`startingWorkspaceLineContent = await this.driverHelper.getDriver().findElement(WorkspaceHandlingTests.READY_TO_READ_WORKSPACE_NAME_LOCATOR).getText();
`
and then make this locator more relative:
`private static READY_TO_READ_WORKSPACE_NAME_LOCATOR: By = By.xpath('//h1[contains(.,'Starting workspace ')]');
`
after it we don't need this field anymore and we can remove it: 
`private static START_WORKSPACE_PAGE_NAME_LOCATOR: By = By.xpath('//div[@class="ui-container"]/div[@class="pf-c-page"]//div[@class="pf-c-content"]/h1');
`

Reply:
Implemented the suggestion. If it's suitable, please resolve this thread and feel free to merge if LGTM, c.c.: @dmytro-ndp

Reply:
If it works, I don't have objections.